source/geod24/concurrency.d

```diff
@@ -1503,6 +1503,152 @@ private:
     assert(x[0] == 5);
 }
 
+/***********************************************************************
+
+    A common interface for objects that can be used in `select()`
+
+***********************************************************************/
+
+public interface Selectable
+{
+    void selectWrite (void* ptr, SelectState sel_state, int sel_id);
+    void selectRead (void* ptr, SelectState sel_state, int sel_id);
+}
+
+/***********************************************************************
+
+    An aggregate to hold neccessary information for a select operation
+
+***********************************************************************/
+
+public struct SelectEntry
+{
+    /// Reference to a Selectable object
+    Selectable selectable;
+
+    /// Pointer to the select data
+    void* select_data;
+
+    this (Selectable selectable, void* select_data)
+    {
+        this.selectable = selectable;
+        this.select_data = select_data;
+    }
+}
+
+public alias SelectReturn = Tuple!(bool, "success", int, "id");
+
+/***********************************************************************
+
+    Block on multiple `Channel`s
+
+    Only one operation is completed per `select()` call
+
+    Params:
+        read_list = List of `Channel`s to select for read operation
+
```

Comment:
boop

Reply:
With no space in between those lines the doc generator did not put each param in its own line.

Reply:
Huh? That sounds like a bug. :/